pkgs/applications/video/obs-studio/default.nix

```diff
@@ -46,19 +51,20 @@ let
 in
 mkDerivation rec {
   pname = "obs-studio";
-  version = "27.2.4";
+  version = "28.0.3";
 
   src = fetchFromGitHub {
     owner = "obsproject";
     repo = "obs-studio";
     rev = version;
-    sha256 = "sha256-OiSejQovSmhItrnrQlcVp9PCDRgAhuxTinSpXbH8bo0=";
+    sha256 = "sha256-+4H1BjEgxqkAEvRyr2Tg3wXutnMvlYQEdT5jz644fMA=";
     fetchSubmodules = true;
   };
 
   patches = [
     # Lets obs-browser build against CEF 90.1.0+
     ./Enable-file-access-and-universal-access-for-file-URL.patch
+    ./Provide-runtime-plugin-destination-as-relative-path.patch
```

Comment:
Does it make more sense to set the variable we're patching in the wrapper (like we set some runtime dependencies in the preFixup phase)? 🤔

Reply:
It would be much cleaner if we could set it there but I don't think it'll work. If I understand qtWrapperArgs properly that set just gets turned into an envvar, but OBS_PLUGIN_DESTINATION isn't a runtime envvar, it's just a cmake variable built into a #define.

See: [Injection from cmake to C](https://github.com/obsproject/obs-studio/blob/277dafab98c087c71deb005cbfaf9ea05be2efc8/libobs/obsconfig.h.in#L16), [Usage from C](https://github.com/obsproject/obs-studio/blob/277dafab98c087c71deb005cbfaf9ea05be2efc8/libobs/obs-nix.c#L57)

Reply:
> OBS_PLUGIN_DESTINATION isn't a runtime envvar

Tried it out and didn't worked for me so this explains it 👍🏻